.github/workflows/coveralls.yaml

```diff
@@ -0,0 +1,31 @@
+name: Coveralls report
+on:
+  push:
+    branches:
+      - main
+  pull_request:
+    paths:
+      - 'subber/**'
+      - 'adder/**'
+      - 'go.mod'
+
+jobs:
+  report:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Wait on tests
+        uses: lewagon/wait-on-check-action@v1.3.1
+        with:
+          ref: ${{ github.sha }}
+          repo-token: ${{ secrets.GITHUB_TOKEN }}
+          wait-interval: 10 # seconds
+          running-workflow-name: report
+          allowed-conclusions: success,skipped,cancelled,failure
+
+      - uses: coverallsapp/github-action@master
+        env:
```

Comment:
Error: Parallel webhook error:Error: {"error":"No build matching CI build number a96768d7d6ebc7816b73dfeefecec738184d7c6e found"}, {"error":"No build matching CI build number a96768d7d6ebc7816b73dfeefecec738184d7c6e found"}

When running github workflows, coveralls workflow thrown error. Can you check this once @mrexox ?

Reply:
Yes, I found an issue in this action repo: https://github.com/lewagon/wait-on-check-action/issues/25. I've applied a recommended, so now is should work correctly.